### ISSUE TITLE
Can't set up the DQ rules for an imported file until you have imported it because the program won't know the field names

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#1693]
### Expected Behavior
I want to set up DQ rules in the DQ Admin screen for a file I am going to import. But I haven't imported it yet, so the program doesn't know what fields are in it (at least the fields that are put in extra_data, which can be almost all of the fields in some cases), so I can't select the fields to set up the DQ rules.

Not sure we can deal with this situation but it's an interesting conundrum ! I encountered it while trying to test the DQ Admin world on a new file that hadn't been imported before with fields that were not standard fields. 
### Actual Behavior 
N/A
### Steps to Reproduce
N/A
### Instance Information
Instance: dev1
SHA: ?? get an error; whatever the SHA is on 8/7/2018 at 6:40 pm PDT

**Comment by nllong** [comment]
Yeah this was a known challenge from the beginning. I think we just need to
update how we recommend people upload their data.

I think the workflow would be to upload a few rows of data to “initialize”
the columns and column mappings. They should also set up the list settings
since that can significantly reduce the inventory list load times and is
much easier to do when there is less data. At that point they can then
upload the larger data file.

**Comment by RDmitchell** [comment]
Yes I think that makes sense. So it's a documentation / workflow process issue rather than a program issue. 

I will assign this to myself and add a "Documentation" label to it.

**Comment by RDmitchell** [comment]
@nllong -- I guess I will document the solution you mention, although it is not a great solution from a user's standpoint.

**Comment by isalanglois** [comment]
@RDmitchell  Where does this ticket stand?

**Comment by RDmitchell** [comment]
@isalanglois -- it is still a problem. Similar to what I was talking about yesterday that you can't rename your matching fields until you have imported your first set of data, where you have to give it one of the canonical matching field names. 

Not sure how to resolve it but we need to rethink how users set up their fields -- right now the program does it from imported data. We may need to have a step that precedes that with some steps where the users can define the fields before they are imported in a file. This needs some design.

**Comment by anchapin** [comment]
I'm going to investigate this to see if I can come up with a better solution since I'm working on the task of making it easier to setup a new organization with a lot of existing data.
